src/test/regress/expected/multi_transaction_recovery.out

```diff
@@ -250,7 +250,7 @@ SELECT pg_reload_conf();
  t
 (1 row)
 
-SELECT pg_sleep(0.1);
+SELECT pg_sleep(1);
```

Comment:
We also do this in multi_mx_transaction_recovery

Reply:
@marcocitus added there too